samples/AuthRemoteFormsAuth/AuthRemoteFormsAuthFramework/Global.asax.cs

```diff
@@ -4,21 +4,27 @@
 using System.Web.Optimization;
 using System.Web.Routing;
 using Microsoft.AspNetCore.SystemWebAdapters;
+using Microsoft.AspNetCore.SystemWebAdapters.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
 
 namespace FormsAuth
 {
     public class Global : HttpApplication
     {
-        void Application_Start(object sender, EventArgs e)
+        protected void Application_Start()
         {
-            SystemWebAdapterConfiguration.AddSystemWebAdapters(this)
-                .AddVirtualizedContentDirectories()
-                .AddProxySupport(options => options.UseForwardedHeaders = true)
-                .AddRemoteAppServer(options =>
-                    {
-                        options.ApiKey = ConfigurationManager.AppSettings["RemoteApp__ApiKey"];
-                    })
-                .AddAuthenticationServer();
+            HttpApplicationHost.RegisterHost(builder =>
+            {
+                builder.AddServiceDefaults();
```

Comment:
One other typical thing that is done when you add aspire to an app is to call abb.MapDefaultEndpoints() which would get things like healthchecks working. That would obviously not go here but just mentioning it in case you want to add that as well.